src/miral/mru_window_list.h

```diff
@@ -29,6 +29,7 @@ class MRUWindowList
 public:
 
     void push(Window const& window);
+    void push_unfocused(Window const& window);
```

Comment:
This is more an "insert_below_top" than a "push"

Reply:
I agree. I thought this naming was more concise but it's likely misleading and/or vague